dev/io.openliberty.security.jakartasec.4.0.internal/bnd.bnd

```diff
@@ -24,11 +24,17 @@ WS-TraceGroup: \
 
 Private-Package: io.openliberty.security.jakartasec.internal.resources.*
 
+-dsannotations: \
+  io.openliberty.security.jakartasec.identitystore.permissions.IdentityStorePermissionServiceImpl
+
+-dsannotations-inherit: true
+
 Export-Package: \
     io.openliberty.security.jakartasec,\
     io.openliberty.security.jakartasec.credential,\
     io.openliberty.security.jakartasec.el,\
     io.openliberty.security.jakartasec.identitystore,\
+    io.openliberty.security.jakartasec.identitystore.permissions,\
```

Comment:
Not sure why you are exporting it.  I don't think there is any reason to.

Reply:
Yeah you need to export it for it to be used with the other ones.  When it was a service it didn't need to be export since it was an impl and you only need the interface to be exported, but now it is the class referenced outside of this bundle so it does need to be exported, so it is the right thing to do.